tests/providers/amazon/aws/sensors/test_s3_upload_session_complete.py

```diff
@@ -0,0 +1,120 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+from unittest import TestCase
+
+from freezegun import freeze_time
+
+from airflow.models.dag import DAG, AirflowException
+from airflow.providers.amazon.aws.sensors.s3_upload_session_complete import S3UploadSessionCompleteSensor
+
+TEST_DAG_ID = 'unit_tests_aws_sensor'
+DEFAULT_DATE = datetime(2015, 1, 1)
+
+
+class TestS3UploadSessionCompleteSensor(TestCase):
+
+    def setUp(self):
+        args = {
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE,
+        }
+        dag = DAG(TEST_DAG_ID + 'test_schedule_dag_once', default_args=args)
+        dag.schedule_interval = '@once'
+        self.dag = dag
+
+        self.sensor = S3UploadSessionCompleteSensor(
+            task_id='sensor_1',
+            bucket_name='test-bucket',
+            prefix='test-prefix/path',
+            inactivity_period=12,
+            poke_interval=10,
+            min_objects=1,
+            allow_delete=False,
+            dag=self.dag
+        )
+
+    def test_reschedule_mode_not_allowed(self):
+        with self.assertRaises(ValueError):
+            S3UploadSessionCompleteSensor(
+                task_id='sensor_2',
+                bucket_name='test-bucket',
+                prefix='test-prefix/path',
+                poke_interval=10,
```

Comment:
```suggestion
                poke_interval=0.1,
```

We want our tests to run quickly!

Reply:
Oh this possibly doesn't matter as it's only run once?

Reply:
Yeah, probably, but I'll update it anyways for future-proofing.